src/k8s-engine/minikube.js

```diff
@@ -109,6 +109,15 @@ class Minikube extends EventEmitter {
       // TODO: Handle the difference between changing version where a wipe is needed
       // and upgrading. All if there was a change.
       args.push("--kubernetes-version=" + this.cfg.version);
+      let memoryInGB = this.cfg.memoryInGB;
+      if (memoryInGB != 2) {
```

Comment:
Any reason to not just always pass in `--memory` to minikube, in case it changes the defaults later?

Reply:
Yes, minikube is broken. The diffs are that the first command specifies `--memory=2g` and the second command doesn't specify any memory setting. Specifying any memory value, even the current one, gives a ❗(error message?) (warning?) (advisory???) message.
```
$ MINIKUBE_HOME=/Users/ericp/Library/Application\ Support/rancher-desktop minikube start -p rancher-desktop --driver hyperkit --container-runtime containerd --interactive=false --kubernetes-version=v1.19.2 --memory=2g
😄  [rancher-desktop] minikube v1.15.1 on Darwin 10.15.3
    ▪ MINIKUBE_HOME=/Users/ericp/Library/Application Support/rancher-desktop
🆕  Kubernetes 1.19.4 is now available. If you would like to upgrade, specify: --kubernetes-version=v1.19.4
✨  Using the hyperkit driver based on existing profile
❗  You cannot change the memory size for an exiting minikube cluster. Please first delete the cluster.
👍  Starting control plane node rancher-desktop in cluster rancher-desktop
^C

 $ MINIKUBE_HOME=/Users/ericp/Library/Application\ Support/rancher-desktop minikube start -p rancher-desktop --driver hyperkit --container-runtime containerd --interactive=false --kubernetes-version=v1.19.2
😄  [rancher-desktop] minikube v1.15.1 on Darwin 10.15.3
    ▪ MINIKUBE_HOME=/Users/ericp/Library/Application Support/rancher-desktop
🆕  Kubernetes 1.19.4 is now available. If you would like to upgrade, specify: --kubernetes-version=v1.19.4
✨  Using the hyperkit driver based on existing profile
👍  Starting control plane node rancher-desktop in cluster rancher-desktop
```

Reply:
Odd, my current minikube (using the vendored one from rd) doesn't seem to complain.

<details><summary>logs</summary>

```
$ ./minikube  start --memory=3g
…
$ ./minikube ssh -- free -h
              total        used        free      shared  buff/cache   available
Mem:          2.9Gi       638Mi       1.4Gi       495Mi       956Mi       1.7Gi
Swap:            0B          0B          0B
$ ./minikube stop
✋  Stopping node "minikube"  ...
🛑  1 nodes stopped.
$ ./minikube start --memory=3g
😄  minikube v1.14.0 on Darwin 10.15.7
✨  Using the hyperkit driver based on existing profile
👍  Starting control plane node minikube in cluster minikube
🔄  Restarting existing hyperkit VM for "minikube" ...
🐳  Preparing Kubernetes v1.19.2 on Docker 19.03.12 ...
🔎  Verifying Kubernetes components...
🌟  Enabled addons: storage-provisioner, default-storageclass
🏄  Done! kubectl is now configured to use "minikube" by default
$ ./minikube ssh -- free -h
              total        used        free      shared  buff/cache   available
Mem:          2.9Gi       620Mi       1.4Gi       495Mi       955Mi       1.7Gi
Swap:            0B          0B          0B
```
</details>

Either way: on initial start, we should still specify `--memory=2g`, because if a future version of minikube defaults to 3g instead, and we did _not_ specify anything, then when the user selects 2 we'll do 3 instead (and when the user selects 3GB we'll specify `--memory=3g` and do the right thing).  If we want to avoid specifying `--memory` when it's already up, we need to do that instead.